geoclue-demo-agent.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
This unit is localtime specific and should be named something like `localtime-geoclue-agent.service`.

Reply:
(i.e., it runs as the localtime user)